app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,28 @@
+<header>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="/"><img src="../assets/logostatic.png" style="width:50px;height: 50px"></a>
```

Comment:
link_to với image_tag nè

Reply:
> link_to với image_tag nè
chưa có dữ liệu thì form_ for vs image gì ông?